hw/ip/rv_timer/data/rv_timer_testplan.hjson

```diff
@@ -9,20 +9,20 @@
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
   entries: [
     {
-      name: random
-      desc: '''RV_TIMER random test performs following steps for number of iterations
+      name: smoke
+      desc: '''RV_TIMER smoke test performs following steps for number of iterations
             - Program zero to CTRL.active* Register(deactivate timer)
             - Program random legal values in CFG*, TIMER_V_*, COMPARE_*, INTR_ENABLE*
             - Program one to CTRL.active* (activate timer)
             - Wait for number of cycles to have mTime>= mTimeCmp
             - Check Interrupt state register and Interrupt signal (scoreboard logic)'''
       milestone: V1
-      tests: ["rv_timer_smoke", "rv_timer_random"]
+      tests: ["rv_timer_smoke"]
```

Comment:
Aah, I am sorry I should have been clear. I prefer to keep this as `random` ('smoke' is then seen as a 'mode' which reduces the scope of the random test to a smaller test).

Reply:
Oh I see! Sorry for misunderstood that. I will change it back :)